### DISCUSSION TITLE
Xstate actions not been assigned after server reloads

### DISCUSSION THREAD

**Discussion by vinodKumarPula** [statelyai/xstate#1958] [Q&A]
I'm using delay transition - after , and the states are persistent. So when server stops and I start again with persistent state. Only the current action will work
Next after actions are not been assigned to actions

Sandbox Details:
https://codesandbox.io/s/xstate-persistent-test-0uw81

**Comment by Andarist** [comment]
Please list actual and expected results - it's hard to dive into this quickly without information like this.

**Reply by vinodKumarPula** [reply]
Hi @Andarist , i'm using xstate persistently. I have delay events specified in  **after** , when the server shutdowns and restart. only the current actions are triggered.After that no actions are assigned
